File: TheProgrammeOfArt/DemoOfChapter2/ContentProviderDemo/app/src/main/java/com/example/contentproviderdemo/application/MyApplication.java
package com.example.contentproviderdemo.application;

import android.app.Application;

import com.example.contentproviderdemo.utils.LogUtils;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
//        LogUtils.i("application start, process name: ");
    }
}

File: LittleDemo/ARouterDemo/app/src/main/java/com/example/arouterdemo/application/MyApplication.kt
package com.example.arouterdemo.application

import android.app.Application
import com.alibaba.android.arouter.launcher.ARouter
import com.example.arouterdemo.BuildConfig

class MyApplication: Application() {

    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) {
            ARouter.openLog()
            ARouter.openDebug()
        }
        ARouter.init(this)
    }

}